Lecture3/Task1/utils/IOFileReader.java

```diff
@@ -0,0 +1,36 @@
+package utils;
+
+import java.io.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class IOFileReader {
+
+	public static void getFileInfo() {
+		System.out.println("Info Written to File:");
+		File listOfProducts = new File("M://GitHub/products.txt");
+		try {
+			BufferedReader getInfo = new BufferedReader(new FileReader(listOfProducts));
+			String productInfo = getInfo.readLine();
+			while (productInfo != null) {
+				Pattern pattern = Pattern.compile("\\{([\\w\\W)])+}");
+				Matcher matcher = pattern.matcher(productInfo);
+				if (matcher.find()) {
+					System.out.println("Guitar object: " + matcher.group(0));
+				}
+				pattern = Pattern.compile("\\w[A-Z]+");
```

Comment:
DO NOT ovverride existing  Pattern pattern  with new value 
move to static final Pattern BLABLABL_PATTERN

Reply:
done